whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -105,6 +118,15 @@ public RpslObject apply(final RpslObject rpslObject) {
         }
     }
 
+    private boolean canAuthenticate(final List<RpslAttribute> attributes){
+        return !attributes.isEmpty() || overrideCredential != null;
+    }
+
+    private boolean isOverrideAuthenticated(final ObjectType objectType){
+        return overrideCredentialValidator != null && overrideCredentialValidator.isAllowedAndValid(isTrusted,
```

Comment:
overrideCredentialValidator is a component, it can never be null?

Reply:
it can be null
FilterAuthFunction is not a component
Please have a look at RpslResponseDecorator 
FILTER_AUTH_FUNCTION -> This setting everything to null
and the is calling apply in line 178

Reply:
FilterAuthFunction should be a component...but I don't want to keep making refactors